src/pages/documentation/platform-assurance/tutorials/versions.md

```diff
@@ -0,0 +1,27 @@
+# Versions Assurance View
+
+## Overview
+
+You may use the Extensions Version view in order to quickly and easily view which Platform extensions you have installed and if they are up to date in a client connected to an Assurance (also known as Project Griffon) session. 
```

Comment:
Since Project Griffon is the old name, I'd go with "(formerly known as Project Griffon)" here instead.